generated/osdep/meson.build

```diff
@@ -0,0 +1,68 @@
+xcrun = find_program('xcrun')
+swift_prog = find_program(run_command(xcrun, '-find', 'swift').stdout().strip())
+
+# Split the returned output based on '.'
+# The last character of the first array member is the major version.
+# We don't care about potential patch/point releases so we only fetch the
+# minor version.
```

Comment:
tbh, i do care for the full version. there were cases when only the full version is able to identify a problem.

what was worng with the original regex? https://github.com/mpv-player/mpv/blob/6265724f3331e3dee8d9ec2b6639def5004a5fa2/waftools/detections/compiler_swift.py#L22-L23

Reply:
Nothing but I didn't want to add another python script to only do a single regex if I didn't have to. I guess I'll run a big long python command instead.